utils/bitcoin.helpers.ts

```diff
@@ -259,6 +293,19 @@ export const calculateFees = (data: Buffer) => {
       BITCOIN_TX.P2WPKH_OUTPUT_VBYTES) *
       BITCOIN_FEES.DEFAULT_REVEAL_FEE_RATE
   );
-  const totalFee = commitFee + revealFee;
-  return { commitFee, revealFee, totalFee };
+
+  const depositFee = await getDepositFee(api);
+
+  const totalFee = commitFee + revealFee + depositFee;
+  return { commitFee, depositFee, revealFee, totalFee };
+};
+
+/**
+ * Calculates the deposit fee for a Bitcoin transaction
+ * @param txFee - Total transaction fee (totalInputValue - totalOutputValue)
+ * @param txVsize - Virtual size of the transaction in vbytes
+ * @returns The calculated deposit fee in satoshis
+ */
+export const calculateDepositFee = (txFee: number, txVsize: number): number => {
+  return Math.ceil((txFee / txVsize) * 68 * 2);
```

Comment:
What are these numbers? I see the number `68` used in a lot of different places, it seems like it's a very important number in the context of Bitcoin, probably worth saving in a constant with a descriptive name.

Reply:
We're no longer using this function, so I deleted it: 7d6fa5e